### PR TITLE
feat: add crude_bronze_nail and adjust others to allow entry to the b…

### PR DESCRIPTION
**Pull request by juur**
#### Summary
Content "Minor changes and additional items to allow progression into bronze age."

#### Purpose of change

Currently, without using iron/steel, progression gets as far as bronze anvil. This patch attempts to unlock subsequent technologies, hopefully keeping balance.

For mods like Innawood this is more of a problem.

#### Describe the solution

Added a crude_bronze_nail (similar to spike nails) and permitted their use with small sandcasting mold. A few other tweaks were needed to allow crafting of bronze_nails without bronze_nails. The next road block is hotcut and a few other items that need steel/iron, which I think are reasonable to block at.

#### Describe alternatives you've considered

No other ideas sprang to mind. I didn't think the 'nail spike' type nails would work as ammo, but they could I guess?

#### Testing

Tested in game with crafting beyond the 'wall' of bronze_nails.

### CODE REVIEW

**Comment by Karol1223** [comment]
> Currently, without using iron/steel, progression gets as far as bronze anvil. This patch attempts to unlock subsequent technologies, hopefully keeping balance.

This is not a concern at all. The goal isn't to allow players to make everything from scratch. If Innawood is facing a problem there, this likely should be adressed in Innawood.

The recipes shouldn't be added to vanilla because the progression doesn't get "far enough". If you have sources that these recipes are valid in reality, then *that's* a reason to add them to vanilla.

File: data/json/items/tool/metalworking.json [hunk 1]
```diff
@@ -1,4 +1,20 @@
 [
+  {
+    "id": "crude_bronze_nail",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "crude bronze nail" },
+    "description": "A crude bronze nail.  Probably not useful for many things due to its size and shape.",
+    "ascii_picture": "nail",
+    "weight": "16 g",
+    "volume": "750ml",
```
Comment (Karol1223) [inline data/json/items/tool/metalworking.json:10]: This is just an absurd density?

**Review by Karol1223 — CHANGES_REQUESTED** [review]

File: data/json/items/tool/metalworking.json (hunk 1 above)
Comment (juur) [reply data/json/items/tool/metalworking.json:10]: Thanks, I'll take a more detailed look.

Comment (juur) [reply data/json/items/tool/metalworking.json]: I think this is OK now, sorry about that.

File: data/json/recipes/other/parts_construction.json [hunk 2]
```diff
@@ -13,6 +13,39 @@
...
+    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 2 } ],
+    "tools": [ [ [ "forge", 20 ], [ "oxy_torch", 20 ] ] ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_redsmithing" } ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "bronze_nail",
+    "id_suffix": "simple",
+    "charges": 6,
```
Comment (Karol1223) [inline data/json/recipes/other/parts_construction.json:37]: Still lots of mass loss here

**Review by Karol1223 — CHANGES_REQUESTED** [review]

File: data/json/recipes/other/parts_construction.json (hunk 2 above)
Comment (juur) [reply data/json/recipes/other/parts_construction.json:37]: How would you solve it then? It seems unlikely a crude forging of nails from 300g of bronzes would result in 300g of nails, there would be many tiny scraps of molten bronze left behind. If the preference is to have nearer 300g in equals 300g out I'll just change it.

Comment (Karol1223) [reply data/json/recipes/other/parts_construction.json:37]: > How would you solve it then? It seems unlikely a crude forging of nails from 300g of bronzes would result in 300g of nails, there would be many tiny scraps of molten bronze left behind. If the preference is to have nearer 300g in equals 300g out I'll just change it.

You are losing 276g/300g of bronze here. This is not "leaving tiny scraps of molten bronze behind".

Comment (juur) [reply data/json/recipes/other/parts_construction.json:37]: ... so your suggestion is?

I can only see three options:

1. Make 100s of nails (not very practical or realistic)
2. Have huge losses
3. Introduce a smaller unit of bronze similar to copper 

What would you be happy with?
Do you have any helpful ideas?

Comment (Karol1223) [reply data/json/recipes/other/parts_construction.json:37]: > What would you be happy with?

More than 6 nails out of a 300g chunk of metal. 6 nails is ridiculously low.

Comment (juur) [reply data/json/recipes/other/parts_construction.json:37]: It's 30 not 6.

Comment (Karol1223) [reply data/json/recipes/other/parts_construction.json:37]: > It's 30 not 6.

You are looking at a different recipe that I am reviewing. You've added two recipes in this PR, and only changed one when I asked you to change them.

Comment (juur) [reply data/json/recipes/other/parts_construction.json]: Updated.

**Pull request merged**